doc/user/content/guides/cdc-postgres.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Change Data Capture (Postgres)"
```

Comment:
Annoying to just use "Postgres" as we use "PostgreSQL" in most docs, but "PostgreSQL" makes this break into a new line and take a bunch of extra space in the menu.

Reply:
We can make the left margin wider if we think this is a general issue, but I'm worried about narrowing the main body of the page more.